src/hotspot/cpu/x86/stubGenerator_x86_64_aes.cpp

```diff
@@ -2181,6 +2181,7 @@ void StubGenerator::aesctr_encrypt(Register src_addr, Register dest_addr, Regist
 
   const Register rounds = rax;
   const Register pos = r12;
+  const Register tail = r13;
```

Comment:
Better to use tail = r15 here. It looks to me that using tail as r13 will cause problems on Windows platform. used_addr is set as r13 in generate_counterMode_VectorAESCrypt() (line 398) for Windows platform and is needed at line 2655 so there is a conflict if we overwrite r13 as tail.

Reply:
Good point. I'll change to `r15` then.